### DISCUSSION TITLE
Busway routing not working as expected

### DISCUSSION THREAD

**Discussion by harto** [valhalla/valhalla#4044] [Q&A]
I was wondering how to troubleshoot a particular bus route.

In this example, I expect the bus to be routed across the bridge: https://valhalla.openstreetmap.de/directions?profile=bus&wps=153.02197158336642%2C-27.47192427383597%2C153.02006721496585%2C-27.472890450151663

The bridge appears to be tagged `highway=busway`: https://www.openstreetmap.org/way/145890014

I expected this to work as of https://github.com/valhalla/valhalla/pull/3413, but I'm not really sure how to troubleshoot. Any guidance would be appreciated 😄

**Comment by kevinkreiser** [comment]
if you try this:

    curl -d '{"locations":[{"lon": 153.021601,"lat": -27.472091}],"verbose":true}' https://valhalla1.openstreetmap.de/locate | jq . | less

you'll see that bus access is turned off! this is probably some strange interplay between `access: no` and `highway: busway`. we can easily make a unit test for this...

lets continue over here: https://github.com/valhalla/valhalla/pull/4045

**Reply by kevinkreiser** [reply]
this was merged and should fix the issue with a fresh build of data
